CONTRIBUTING.md

```diff
@@ -111,3 +115,60 @@ A PR can only be merged into master by a maintainer, if all of these conditions
 * It has been approved by at least two maintainers. If it was a maintainer who opened the PR, only one extra approval is needed.
 * It has no requested changes.
 * It is up to date with current master.
+
+### Release process
+
+Maintainers need to perform the following actions **in the order described here** to push out a release.
+
+#### Verify the version in [DESCRIPTION](./DESCRIPTION) file
+
+Create a new PR and merge it, if the version in the [DESCRIPTION](./DESCRIPTION) file is not good.
+
+#### Prepare the GitHub Release Notes
+
+- Open [github releases](https://github.com/process-analytics/bpmn-visualization-R/releases)
+- Create a new draft release and name it `Next` (the name is not relevant and will be replaced automatically later).
+  This ensures that development can continue without impacting the writing of the content of the in progress release. That way,
+  if a PR is merged, `release-drafter` will update the `Next` draft release keeping the in-progress release untouched.
+- The draft release for the newly tagged version should already exist:
+   - [release-drafter](https://github.com/release-drafter/release-drafter) creates or updates draft release for the
+     next version each time a pull request is merged to the `master` branch.
+   - create a new release if it is missing or rename the existing one to match.
+- Assign the new tag as release target and save the draft (this should have already been managed by `release-drafter`)
+- Ensure that `This is a pre-release` is unchecked (except if we are releasing alpha, beta, rc, ...)
+- In the release description (check previous releases as a source of inspiration), put screenshots/gif of the new features.
+- At any time, you can save the draft.
+
+#### Release on GitHub (tag)
+
+- Open the draft release note in [github releases](https://github.com/process-analytics/bpmn-visualization-R/releases)
+- Verify the tag version as the same as the release note.
+- Published the release **only when you are done** with the release content. 
+
+#### Publish the new version on CRAN
+
+- Pull the new version created by the release workflow: `git pull origin`
+- Ensure you are on the last tag locally: `git checkout vX.Y.Z`
+- Open the project in the R Studio
+- On the **Build** tab, click on **More** > **Build Binary Package**
+<p align="center"><img title="Build Binary Package" src="./doc/r-studio-binary-build.png" alt="Build Binary Package"></p>
```

Comment:
⚠️ we have to decide where to store the image used by the internal documentation. In #45, I put the readme image in a `img` folder, but we should only have one folder.
I don't know if `doc` is ok for R project, especially when we will implement #40. But for now, I am ok to use the `doc` folder.

@csouchet do we keep the `doc` folder? If so, I will update the other PR.

Reply:
I don't have preferences. I just reproduce what we did on bpmn-visualization-js.

Reply:
ok, let's start with the `doc` folder

Reply:
Integrated in #45 which has been merged in the `main` branch